pkgs/applications/video/handbrake/default.nix

```diff
@@ -110,8 +105,10 @@ stdenv.mkDerivation rec {
   ];
 
   # NOTE: 2018-12-27: Check NixOS HandBrake test if changing
+  # NOTE: The -ldl flag was fixed upstream for a release after 1.3.0
```

Comment:
Could you move it inside the list, just before the relevant line?

Reply:
Done.